src/content/reference/react-dom/client/index.md

```diff
@@ -1,22 +1,23 @@
 ---
-title: Client React DOM APIs
+title: API klien React DOM
 ---
 
 <Intro>
 
-The `react-dom/client` APIs let you render React components on the client (in the browser). These APIs are typically used at the top level of your app to initialize your React tree. A [framework](/learn/start-a-new-react-project#production-grade-react-frameworks) may call them for you. Most of your components don't need to import or use them.
+
+API `react-dom/client` memungkinkan Anda merender komponen React di sisi klien (di peramban). Biasanya API ini digunakan pada level atas aplikasi React untuk menginisialisasi pohon React. Sebuah [*framework*](/learn/start-a-new-react-project#production-grade-react-frameworks) mungkin memanggilnya untuk Anda. Biasanya komponen Anda tidak perlu mengimpor atau menggunakannya.
```

Comment:
Need to fix all `merender` to `me-*render*`. @r17x @mazipan @resir014 @zainfathoni can help to check.

```suggestion
API `react-dom/client` memungkinkan Anda me-*render* komponen React di sisi klien (di peramban). Biasanya API ini digunakan pada level atas aplikasi React untuk menginisialisasi pohon React. Sebuah [*framework*](/learn/start-a-new-react-project#production-grade-react-frameworks) mungkin memanggilnya untuk Anda. Biasanya komponen Anda tidak perlu mengimpor atau menggunakannya.
```

Reply:
Updated, thanks